TTPChallenge/AllTechTypesVC.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  AllTechTypesVC.swift
+//  TTPChallenge
+//
+//  Created by Mirim An on 2/15/17.
+//  Copyright © 2017 TeamMDC. All rights reserved.
+//
+
+import UIKit
+
+class AllTechTypesVC: UIViewController {
+
+    @IBOutlet var allTechJobsButtons: [UIButton]!
```

Comment:
Also, I think having that array leads to a retain cycle. I don't think the buttons will die with the view. The view has a strong hold on the array, the array holds the buttons strongly, the buttons hold the array strongly so when the view dies, the array and buttons never let go...